Context/RestApiContext.php

```diff
@@ -526,6 +526,30 @@ public function theResponseCollectionShouldCountItems($expectedValue)
         return;
     }
 
+    /**
+     * When response JSON is a collection (array), it checks if there is field with value.
+     *
+     * Example: Then the response collection should have field "name" with value "abcdef"
+     *
+     * @Then the response collection should have field :fieldName with value :expectedValue
```

Comment:
@Then at least one of the collection items should have field :fieldName with value :expectedValue